soh/src/overlays/actors/ovl_Oceff_Wipe/z_oceff_wipe.c

```diff
@@ -80,10 +80,12 @@ void OceffWipe_Draw(Actor* thisx, PlayState* play) {
 
     OPEN_DISPS(play->state.gfxCtx);
 
+    int fastOcarinaPlayback = CVarGetInteger("gFastOcarinaPlayback", 0);
```

Comment:
```suggestion
    int fastOcarinaPlayback = (CVarGetInteger("gFastOcarinaPlayback", 0) != 0);
```
doing this will make it always 1 or zero, so we can treat it as a bool later (realistically we don't need to do this, as we should only ever get a 1 or zero from the cvar)